ush/python/pygfs/task/archive.py

```diff
@@ -417,3 +438,151 @@ def replace_string_from_to_file(filename_in, filename_out, search_str, replace_s
         replace_string_from_to_file(in_track_p_file, out_track_p_file, "AVNO", pslot4)
 
         return
+
+    @staticmethod
```

Comment:
Why is this static? Especially as a "hidden" method.

Reply:
Force of habit...
```suggestion
```